src/hooks/useUSDCPrice.ts

```diff
@@ -45,7 +46,8 @@ export default function useUSDCPrice(currency?: Currency): Price<Currency, Token
       const { numerator, denominator } = v2USDCTrade.route.midPrice
       return new Price(currency, stablecoin, denominator, numerator)
     } else if (v3USDCTrade.trade) {
-      const { numerator, denominator } = v3USDCTrade.trade.route.midPrice
+      // TODO(judo): is looking at a single midprice enough?
+      const { numerator, denominator } = v3USDCTrade.trade.routes[0].midPrice
```

Comment:
This is using the old client side routing logic right? If so I don't understand the TODO since the old logic doesn't do split routes

If we change this to use the API / client side router I could see us having this issue (though we could use maxSplits = 1 when getting this price to get around it)